packages/rancher-windows-gmsa/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name}}
+  namespace: {{.Release.Namespace}}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ .Release.Name }}
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - linux

```

Comment:
We should either replace this with the node-selector helper template listed above or create a new helper function for `linux-node-affinity` that takes in overrides and use that here. That way we can standardize on templates to get stuff working on Windows.

Reply:
We had discussed this yesterday. I would like to hold off and making this change until the final approach for how to handle this is implemented. I will create a GitHub issue to track.